index.js

```diff
@@ -51,27 +44,38 @@ function uglifyify(file, opts) {
     )
 
     debug = opts.sourceMap !== false && (debug || matched)
-    opts  = extend({}, {
+
+    var thisopts  = extend({}, {
```

Comment:
do you want to rename this to `_opts`, so we can distinguish that this is uglify's opts we are gonna use for uglify only. `thisopts` just seems off for me. but definitely agree that we should create a new obj